packages/cfd/src/features/Containers/trade-modals/other-cfds-trade-modal.tsx

```diff
@@ -78,8 +79,8 @@ const OtherCFDsTradeModal = ({
         const platform_name = 'cTrader';
 
         return (
-            <div className='cfd-trade-modal__login-specs-item'>
-                <Text className='cfd-trade-modal--paragraph'>
+            <div className='trade-modal__login-specs-item'>
+                <Text className='trade-modal--paragraph'>
```

Comment:
better to fallow BEM: 
"trade-modal" is the block.
"trade-modal__login"  is the element.
"trade-modal__download-center" is a modifier for the download element.
```suggestion
                <Text className='trade-modal__login'>
                         <Text className='trade-modal--login--paragraph'>
```

Reply:
this is referring to the text in download-center , rather than positioning download__text at center